### ISSUE TITLE
Update to React 16 and current Router for Rails config?

### ISSUE THREAD

**Issue by jarmstro0** [LaunchAcademy/make_it_so#49]
Several of us found during individual projects that the Node packs we wanted to install had a dependency to React 16.

**Comment by dpickett** [comment]
yeah for sure! We're in the process of upgrading our curriculum to reflect the latest version of router. Once that is set, we can then upgrade `make_it_so` accordingly.

**Comment by CassandraNewell** [comment]
The React upgrade is almost done! I renamed this issue to focus on the React Router upgrade, since that still needs doing.
